build-system/babel-config/pre-closure-config.js

```diff
@@ -53,6 +53,8 @@ function getPreClosureConfig() {
         // Imports that are not needed for valid transformed documents.
         '../build/ampshared.css': ['cssText', 'ampSharedCss'],
         '../build/ampdoc.css': ['cssText', 'ampDocCss'],
+        // Srcset fallbacks aren't needed in ESM builds
+        '../src/utils/img': ['guaranteeSrcForSrcsetUnsupportedBrowsers'],
```

Comment:
I really dislike how these are tied to the import site. If I rename the local import, or move the file to a new location, then this will break. It's not obvious in the file that these have consequences.

Instead, how about we use the `IS_ESM` variable in the imports? Eg, we allow `guaranteeSrcForSrcsetUnsupportedBrowsers` to be imported, but inside the function itself, it has a `if (IS_ESM) { return }`.